recipe/meta.yaml

```diff
@@ -16,10 +16,11 @@ build:
 
 requirements:
   host:
-    - python >=3.5
+    - python >=3.6
     - pip
   run:
-    - python >=3.5
+    - python >=3.6
+    - typing_extensions
```

Comment:
Only needed for `<py38` but if you have a selector then it can't be a noarch package. Is it ok to install when it's not needed so you can keep the recipe noarch?

https://github.com/django/asgiref/blob/34d1602c90ff1f5f0ca70919bc9c6a268285f80a/setup.cfg#L34